AmericanExpress/src/main/java/com/braintreepayments/api/AmericanExpressAnalytics.java

```diff
@@ -0,0 +1,21 @@
+package com.braintreepayments.api;
+
+enum AmericanExpressAnalytics {
+
+    // Get Rewards Balance Events
```

Comment:
The comments in the iOS files are a little inconsistent. [This doc](https://paypal.sharepoint.com/:x:/r/sites/BTSDK/_layouts/15/Doc.aspx?sourcedoc=%7BAAA7C353-5744-4F21-AC7D-DF85D8605817%7D&file=Northstar%20Analytics%20Events.xlsx&action=default&mobileredirect=true&cid=1b0afccd-68ab-448f-9760-60ed14df9811) is our source-of-truth for events. The colored ones (red, blue, green, orange) are the ones we're using to calculate conversion rates, whereas the black ones are superfluous / extra detail we're capturing in case we need to deep-dive on an issue.

Reply:
The intention with the Marks/comments on iOS was to make it clear which 3 or 4 events were the really important ones for "conversion" calculation. I think the [SEPA file](https://github.com/braintree/braintree_ios/blob/main/Sources/BraintreeSEPADirectDebit/BTSEPADirectAnalytics.swift) probably does it best

Reply:
My bad - was looking at an iOS PR instead of that doc - updated now. I removed some of the internal comments where all the events are conversion events an clarified others.